src/sentry/testutils/cases.py

```diff
@@ -646,21 +646,16 @@ def detect_performance_problems_interceptor(
                     perf_problem.fingerprint = fingerprint
             return perf_problems
 
-        with (
-            mock.patch(
-                "sentry.issues.ingest.send_issue_occurrence_to_eventstream",
-                side_effect=send_issue_occurrence_to_eventstream,
-            ) as mock_eventstream,
-            mock.patch(
-                "sentry.event_manager.detect_performance_problems",
-                side_effect=detect_performance_problems_interceptor,
-            ),
-            mock.patch.object(
-                issue_type, "noise_config", new=NoiseConfig(noise_limit, timedelta(minutes=1))
-            ),
-            override_options(
-                {"performance.issues.all.problem-detection": 1.0, detector_option: 1.0}
-            ),
```

Comment:
This was being auto-formatted, but the pre-commit hook would fail for it the following error:
```
src/sentry/testutils/cases.py - Could not parse ast

        Traceback (most recent call last):
          File "/Users/schew/.cache/pre-commit/repoyv1rzlad/py_env-python3/lib/python3.8/site-packages/pre_commit_hooks/debug_statement_hook.py", line 56, in check_file
            ast_obj = ast.parse(f.read(), filename=filename)
          File "/Users/schew/.pyenv/versions/3.8.13/lib/python3.8/ast.py", line 47, in parse
            return compile(source, filename, mode, flags,
          File "src/sentry/testutils/cases.py", line 653
            ) as mock_eventstream,
              ^
        SyntaxError: invalid syntax
```

I committed it originally using the `--no-verify` flag, but this means that anyone editing this file will also run into the same error. I managed to avoid the auto-formating with auto-save on vscode and switch it back for now